src/components/study-manager.js

```diff
@@ -480,6 +483,8 @@ const StudyManager = ({ onClick }) => {
 
     const classes = useStyles();
 
+    const { enqueueSnackbar } = useSnackbar();
+
```

Comment:
Why we not use commons-ui snackbar for this ?

Reply:
I'm using snackbar from commons-ui : see app.js where SnackbarProvider is imported and used

See demo/src/app.js in commons-ui, where there is an exemple of snackbar use

Reply:
ok